packages/core/src/components/side-pane-group/side-pane-group.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Component, Element, h, Host, Prop, State } from '@stencil/core';
+import { applicationLayoutService } from '../utils/application-layout';
+import { Breakpoint, matchBreakpoint } from '../utils/breakpoints';
+
+@Component({
+  tag: 'ix-side-pane-group',
+  styleUrl: 'side-pane-group.scss',
+  shadow: true,
+})
+export class SidePaneGroup {
+  @Element() hostElement: HTMLIxSidePaneGroupElement;
+
+  /**
+   * Determines if the side panes behave inline
+   * Behaviour with string?
+   */
+  @Prop() inline: boolean = false;
+
+  /**
+   * Determines if the side panes behave floating
+   */
+  @Prop() floating: boolean = false;
+
+  /**
+   * Choose the variant of the panes
+   */
+  @Prop() variant: '1' | '2' = '2';
+
+  /**
+   * Supported layouts
+   */
+  @Prop() breakpoints: Breakpoint[] = ['sm', 'md'];
```

Comment:
Should not be provided as prop see review comment at useage of application layout service